pkgs/build-support/cc-wrapper/add-flags.sh

```diff
@@ -10,6 +10,7 @@ var_templates_list=(
     NIX+CXXSTDLIB_COMPILE
     NIX+CXXSTDLIB_LINK
     NIX+GNATFLAGS_COMPILE
+    NIX+HARDENING_ENABLE
```

Comment:
Per https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/cc-wrapper/cc-wrapper.sh#L20-L23 you should put a variable used by both in either just `bintools-wrapper` or both wrappers, never just `cc-wrapper`.

Reply:
I've removed this line in the latest commit.